srcs/app/http/HttpRequest.cpp

```diff
@@ -206,6 +206,10 @@ bool	HttpRequest::IsValidPath_(const std::string &path) const {
 	return !path.empty() && path[0] == '/';
 }
 
+bool	HttpRequest::IsValidDecodedPath_(const std::string &path) const {
```

Comment:
I already mentioned that I don't mind this much but if we're going to do, we may as well be strict about it.

We don't accept "/..", but what about "../"?

Reply:
Great point.
Also, the path could be a filename starting with "..",
for example, "/.....name", and this should be valid.

Reply:
Great point! Also, the path could be a filename starting with "..",
for example, "/.....name", and this should be valid.

Reply:
@Damrod, the last commit should fix these cases.